client/features/checkout/AddressForm.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import TextField from "@material-ui/core/TextField";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Checkbox from "@material-ui/core/Checkbox";
+
+function AddressForm() {
+  return (
+    <React.Fragment>
+      <Typography variant="h6" gutterBottom>
+        Shipping address
+      </Typography>
+      <Grid container spacing={24}>
```

Comment:
spacing values should never be higher than 10. What were you looking to achieve here ?